### ISSUE TITLE
Expand the root node cause expand of all the children

### ISSUE THREAD

**Issue by blackat** [angular-ui-tree/angular-ui-tree#712]
Hi,
my root node is collapsed when the page is loaded, when I click on the root node I would like to expand just one level and not all the children and children of the children.

I use as command to expand `arg.$nodeScope.toggle();` is this the right one?

Thanks a lot for the attention.

**Comment by blackat** [comment]
Found the trick! I have added `data-collapsed="true"``to the `<li>` item in the template and it works like a charm.

```javascript
<ol ui-tree-nodes ng-model="nodes">
    <li class="wkfsfrc-button-bar"
         data-collapsed="true"
         ng-repeat="node in nodes"
         data-drag-delay="1000"
         ui-tree-node
         ng-include="'nodes_renderer.html'">
    </li>
</ol>
```

I have seen other people had the same issue, could you improve the documentation with a small portion of code and a bit of explanation?

Thanks a lot for your work :wink:

**Comment by Voles** [comment]
@blackat glad you found the solution? Are you able to create a PR to improve the docs?

**Comment by blackat** [comment]
Yep sure, I will put a small plunker if you want or just a bit of html and js code, as you prefer.

**Comment by blackat** [comment]
Done it! There is an issue with the coverage, the CI is fine but the coverage 

> Some checks were not successfu

have I done something wrong?

**Comment by jacobscarter** [comment]
Will close when PR #716 is merged
